frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Edit_HPA_Action.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Edit HPA action
+	As a user, I want to remove the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment/deployment-config 
+    And user has assigned values to cpu and memory value in resource limit section of advanced option
+    And user is in topology
+
+@regression
+Scenario: Edit HPA action form view
+   Given user has a workload with HPA assigned to it
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Edit Horizontal Pod Autoscaler option 
+   And user sees Edit Horizontal Pod Autoscaler form opens
+   And user sees values are prefilled
+   And user checks the name value but can not edit it
+   And user checks and edit the value for maximum and minimum pods
+   And user checks and edit the cpu value and memory value
+   And user clicks on save
+   And user is taken to topology
+   And user opens the sidebar
+   And user can see Horizontal Pod Autoscalers section
```

Comment:
```suggestion
   And user sees Horizontal Pod Autoscalers section
```

Reply:
On what pretext you select `sees` or `can see`. Is it that when we are writing for then it will be `can see` and when writing for when part it will be `sees`

Reply:
yes.